_source/_data/authors.yml

```diff
@@ -802,3 +802,33 @@ alina-yurenko:
   web: https://dev.to/alina_yurenko
   twitter: https://twitter.com/alina_yurenko
   bio: I ❤️ all things tech, open source and community. Developer 🥑 for @GraalVM at @OracleLabs, helping @jhipster, ex @googledevgroups
+  
+alex-doukas:
+  full_name: "Alex Doukas"
```

Comment:
Quotes aren't needed around these values. If you look at the others, they work just fine without quotes.